stdlib/public/core/Character.swift

```diff
@@ -355,7 +355,7 @@ public struct Character :
 
 extension Character : CustomStringConvertible {
   public var description: String {
-    return String(self)
+    return String(describing: self)
```

Comment:
This method should not go through reflection either, there are better, more performant ways to do this.  (And wouldn't it call itself in fact?)